File: Data_Structure_and_Algorithms/1.Arrays/problem/3.Remove_duplicates_from_Sorted_Array.py
arr = [1, 2, 3, 4, 4, 4, 5, 6, 7, 7, 7, 8]
arr_complete = []
for i in range(0, len(arr), 1):
    # print("checking i = ", arr[i])
    check = bool(False)
    for j in range(i+1, len(arr), 1):
        # print("checking j = ", arr[j]);
        if (arr[i] == arr[j]):
            check = bool(True); break
    if check == False:
        arr_complete.append(arr[i])

for i in range(0, len(arr_complete), 1):
    print(arr_complete[i], end=' ')

File: Data_Structure_and_Algorithms/1.Arrays/problem/3.Remove_duplicates_from_Sorted_Array.cpp
#include <iostream>
#include <vector>
int main()
{
    std::vector<int> arr = {1, 2, 2, 3, 4, 4, 4, 5, 5};
    std::vector<int> arr_complete = {};
    for (size_t i = 0; i < arr.size(); i++)
    {
        bool check = false;
        for (size_t j = i+1; j < arr.size(); j++)
        {
            if (arr[i] == arr[j])
            {
                check = true; break;
            }
        }
        if (check == false)
        {
            arr_complete.push_back(arr[i]);
        }
    }
    for (size_t i = 0; i < arr_complete.size(); i++)
    {
        std::cout << arr_complete[i] << ' ';
    }
    return 0;
}